crypto/Kconfig

```diff
@@ -34,6 +34,28 @@ if NRF_CC3XX_PLATFORM
 config HW_CC3XX_INTERRUPT
 	bool "Use interrupt version of nrf cc3xx platform library" if !(CC3XX_BACKEND || NRF_CC310_BL)
 
+choice CC310_LOCK_VARIANT
+	prompt "CC310 lock variant"
+	default CC310_MUTEX_LOCK
+
+config CC310_MUTEX_LOCK
+	bool "Use mutex variables for mutually exclusive access to CC310 resources"
+	help
+	  Locking by mutex is the most flexible solution for providing mutually
```

Comment:
Enables exclusive access to the Arm CryptoCell HW using an RTOS mutex that has priority inheritance. The mutex lock is safe across threads and/or interrupts with different priority. Using an RTOS mutex comes at a cost.